pkg/vm/process/types.go

```diff
@@ -72,6 +72,9 @@ type SessionInfo struct {
 	Host         string
 	Role         string
 	ConnectionID uint64
+	AccountId    uint32
```

Comment:
Why do you need to add this information?

Reply:
Because the operation on the table needs to use "AccountId"